scripts/lib/libvirt/templates/admin-node.xml

```diff
@@ -5,8 +5,7 @@
   <vcpu>$adminvcpus</vcpu>
   <os>
     <type arch='$march' machine='$machine'>hvm</type>
-$osloader
-    <nvram>/var/lib/libvirt/$cloud-admin_VARS.fd</nvram>
+    $osloader
```

Comment:
we might not want to indent it because it would produce trailing whitespace when it expands to empty..